java/daikon/dcomp/DCInstrument.java

```diff
@@ -1901,7 +2113,7 @@ InstructionList handleInvoke(InvokeInstruction invoke) {
       // could detect functional interfaces in a manner similar to the Java
       // compiler, but for now we will go with this simpler method.
       //
-      // Note that to simplfy our code we set the access flags for a functional
+      // Note that to simplify our code we set the access flags for a functional
       // interface to ANNOTATION in our class_access_map.
```

Comment:
I suggest renaming `class_access_map` to either `classToAccessFlags` or even just `accessFlags`, which is what it contains.